CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## Upcoming
+
+### Design changes
+
+- Select hover state is now `Blue200`
```

Comment:
No need to specify every color change, just write that design have been updated and in what way.

Also, wrap component names with template string quotes.

Reply:
Fixed!